package.json

```diff
@@ -7,14 +7,20 @@
     "dev": "vite",
     "build": "tsc && vite build",
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
-    "preview": "vite preview"
+    "preview": "vite preview",
+    "build-chrome": "cp manifest.chrome.json ./public/manifest.json && tsc && vite build",
+    "build-firefox": "cp manifest.firefox.json ./public/manifest.json && tsc && vite build",
+    "submit-chrome": "npx chrome-webstore-upload upload --source ./dist --extension-id $CHROME_EXTENSION_ID --client-id $CHROME_CLIENT_ID --client-secret $CHROME_CLIENT_SECRET --refresh-token $CHROME_REFRESH_TOKEN",
+    "submit-firefox": "npx web-ext sign --channel=listed --api-key $FIREFOX_EXT_API_KEY --api-secret $FIREFOX_EXT_API_SECRET --source-dir ./dist"
   },
   "dependencies": {
```

Comment:
Unclear to me what best practice is for dependencies v devDependencies in as case like this one where the output is a compiled artefact.

Reply:
Since we are using `npm ci` in the pipeline, all packages from devDependencies and dependencies will be installed, so it is still safe even if some packages are in devDependencies. Therefore I don't think we need to change these.